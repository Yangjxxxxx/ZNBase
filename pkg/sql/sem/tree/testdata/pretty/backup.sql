-- DUMP
-- DATABASE bank TO SST 'gs://acme-co-backup/database-bank-2017-03-29-nightly'
-- AS OF SYSTEM TIME '-10s'
-- INCREMENTAL FROM 'gs://acme-co-backup/database-bank-2017-03-27-weekly', 'gs://acme-co-backup/database-bank-2017-03-28-nightly'
-- WITH revision_history
DUMP
DATABASE
	bank
TO SST
	'gs://acme-co-backup/database-bank-2017-03-29-nightly'
AS OF SYSTEM TIME
	'-10s'
INCREMENTAL FROM
	'gs://acme-co-backup/database-bank-2017-03-27-weekly',
	'gs://acme-co-backup/database-bank-2017-03-28-nightly'
WITH
	revision_history