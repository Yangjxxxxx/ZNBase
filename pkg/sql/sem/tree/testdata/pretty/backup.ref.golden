// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
DUMP
DATABASE
	bank
TO
 SST
	'gs://acme-co-backup/database-bank-2017-03-29-nightly'
AS OF SYSTEM TIME
	'-10s'
INCREMENTAL FROM
	'gs://acme-co-backup/database-bank-2017-03-27-weekly',
	'gs://acme-co-backup/database-bank-2017-03-28-nightly'
WITH
	revision_history

113:
-----------------------------------------------------------------------------------------------------------------
DUMP
DATABASE
	bank
TO
 SST
	'gs://acme-co-backup/database-bank-2017-03-29-nightly'
AS OF SYSTEM TIME
	'-10s'
INCREMENTAL FROM
	'gs://acme-co-backup/database-bank-2017-03-27-weekly', 'gs://acme-co-backup/database-bank-2017-03-28-nightly'
WITH
	revision_history

252:
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
DUMP DATABASE bank TO SST 'gs://acme-co-backup/database-bank-2017-03-29-nightly' AS OF SYSTEM TIME '-10s' INCREMENTAL FROM 'gs://acme-co-backup/database-bank-2017-03-27-weekly', 'gs://acme-co-backup/database-bank-2017-03-28-nightly' WITH revision_history


